oktoberfest/runner.py

```diff
@@ -144,14 +152,16 @@ def _annotate_and_get_library(spectra_file: Path, config: Config, tims_meta_file
 def _get_best_ce(library: Spectra, spectra_file: Path, config: Config):
     results_dir = config.output / "results"
     results_dir.mkdir(exist_ok=True)
-    if (library.obs["FRAGMENTATION"] == "HCD").any():
-        server_kwargs = {
-            "server_url": config.prediction_server,
-            "ssl": config.ssl,
-            "model_name": config.models["intensity"],
-        }
+    if (library.obs["FRAGMENTATION"].isin(["CID", "HCD"])).all():
```

Comment:
this is wrong. Here, CE calibration is performed. This is only necessary when dealing with HCD data, since for CID, we never observed any change in the intensity of individual fragment peaks of spectra. The reason why this is "weirdly written" here, is that oktoberfest originated from a bunch of loose scripts back in the days, when a mix of HCD and CID data could be submitted, hence the test for `all`.
Technically, we should only ever have either HCD, or CID, unless one would use a model that can deal with both. I.e., `FRAGMENTATION` is a parameter that should be the same for every PSM, thus, could be moved into the `.uns` object of the anndata object in the future. But since `anndata` is allowing to convert columns with few different variables to factors / categoricals, this is already kinda ok as it is stored efficiently in the `.obs` df. 
Please reason why you added CID here.

Reply:
fixed in 68e65a66d6ff820fe30084817fb51260256e2bce